docs/en/advanced/Performance-Tuning.md

```diff
@@ -217,9 +217,9 @@ and immediately returns the requested data to the client.
 The worker will asynchronously continue to read the remainder of the block without blocking the client request.
 
 The number of asynchronous threads used to finish reading partial blocks is set by the
-`alluxio.worker.network.netty.async.cache.manager.threads.max` property, with a default value of `512`.
+`alluxio.worker.network.async.cache.manager.threads.max` property.
 When large amounts of data are expected to be asynchronously cached concurrently, it may be helpful
-to reduce this value to reduce resource contention.
+to increase this value to handle a higher workload.
```

Comment:
I think we should mention the tradeoffs of changing this value. Currently it sounds like increasing it is a strict improvement.

Reply:
Added a new sentence after with caveat